src/contexts/debugging-state/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { createContext, useContext, useState } from 'react'
+import { DebuggingStateProviderProps } from './types'
+
+const DebuggingStateContext = createContext<$TSFixMe>(undefined)
```

Comment:
Do we want to type this? Or just leave it as `any`?

Reply:
Oh I forgot about this! I think we want to do `Record<string, unknown>`?

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/1338/commits/5c137c2782eafd3865c27689a26faddeccadfec7